packages/lesswrong/components/users/DialogueMatchingPage.tsx

```diff
@@ -1270,14 +1270,15 @@ export const DialogueMatchingPage = ({classes}: {
 
   const userDialogueUsefulData: UserDialogueUsefulData = data?.GetUserDialogueUsefulData;
 
-  const matchedUsers: UsersOptedInToDialogueFacilitation[] | undefined = matchedUsersResult?.GetDialogueMatchedUsers;
+  const matchedUsersWithSelf: UsersOptedInToDialogueFacilitation[] | undefined = matchedUsersResult?.GetDialogueMatchedUsers
```

Comment:
All of this really should be done at the DB query level.  Otherwise you'll be playing whack-a-mole later